README.md

```diff
@@ -22,11 +22,14 @@ days, and which days are holidays.
 ```ruby
 calendar = Business::Calendar.new(
   working_days: %w( mon tue wed thu fri ),
-  holidays: ["01/01/2014", "03/01/2014"]
+  holidays: ["01/01/2014", "03/01/2014"],    # array items are either parseable date strings, or real Date objects
+  working_dates: ["08/01/2014"]
```

Comment:
It's probably not worth including this in the first example as this is an advanced feature. I'd talk about it further down in the readme.